deletefb/tools/env_variables.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+import re
+import typing as ty
+
+from types import SimpleNamespace
+
+
+class EnvVar:
```

Comment:
Could you make it into an attrs class, similar to the other ones in https://github.com/weskerfoot/DeleteFB/blob/master/deletefb/types.py ?